salt/config/__init__.py

```diff
@@ -944,6 +944,9 @@ def _gather_buffer_space():
     # Always generate minion id in lowercase.
     'minion_id_lowercase': bool,
 
+    # Remove this single domain (e.g. 'foo.org') in a generated minion id.
+    'minion_id_remove_domain': six.string_types,
```

Comment:
```
 'minion_id_remove_domain': (six.string_types, bool)
```

Reply:
ok

Reply:
@markuskramerIgitt it is just a list of _possible_ types. So just like @DmitryKuzmenko suggested.